docs/experiments/developing_an_experiment.md

```diff
@@ -0,0 +1,116 @@
+# Test Pilot Recommended Development Process for Experiments
+
+The following process has proven to be the most streamlined in the Test Pilot
+world.  You're not required to follow it, but if you do things will probably
+move more smoothly and it'll be easier to get help for your experiment because
+it's a familiar process.
+
+## Sprints
+We're fans of sprinting.  The standard two weeks is a good balance of feature
+development and shipping.
+
+## Communication
+Yay meetings. :)
+
+- Pick Slack or IRC.  Having a channel on each just makes it harder to decide
+  where to talk.  Now that contributors can join Slack, most experiments choose
+  that.
+- Schedule stand-ups at least twice a week.  Most experiments can fit a new-bug
+  triage into the stand-up if they do it consistently so the bugs don't build
+  up.
+- Schedule a sprint planning meeting.
+- Schedule a product and analysis meeting.  This meeting is focused on
+  reviewing the analytics and talking about actions we should take based on
+  them.
+- Every Test Pilot experiment has a [forum on
+  discourse](https://discourse.mozilla.org/c/test-pilot).  Be sure to visit
+  regularly and engage with your users.
+
+
```

Comment:
Nitpicking: newlines aren't consistent.

Reply:
Looks like a plain old word-wrap since the markdown link has no spaces in it. Could neaten it up with a footnote-style link. e.g.:
```markdown
- Every Test Pilot experiment has a [forum on discourse][forum].  Be sure
  to visit regularly and engage with your users.

[forum]: https://discourse.mozilla.org/c/test-pilot
```

Reply:
I'll update this and let the lines with links go past 80 chars.  I do prefer footnote links, but I don't think we use them elsewhere.